src/scripts/fetch_package_names.js

```diff
@@ -65,3 +105,98 @@ function fetchPackageNames(options, cb) {
+
+      request(url, function(err, response, body) {
+        if (err) {
+          return cb(err);
+        }
+
+        if (body && file === 'README.md') {
+          body = body
+            // Remove all but the description from lead-in (XXX: Optional `\/?` should be unnecessary)
+            .replace(/[^]*?<div align="center">([^]*?)<\/div>/, (match, content) => {
+              let parsed = content.match(/<p>([^]*?)<\/?p>/);
+              return parsed ? parsed[1] : '';
+            })
+            // Replace lone h1 formats
+            .replace(/<h1.*?>.+?<\/h1>/, '')
+            .replace(/# .+/, '')
+            // Resolve anchor hrefs to avoid broken relative references in the docs
+            // Examples:
+            // - [click here](LICENSE) --> [click here](https://raw.githubusercontent.com/user/repository/branch/LICENSE)
+            // - [click here](./LICENSE) --> [click here](https://raw.githubusercontent.com/user/repository/branch/LICENSE)
+            // - [click here](#LICENSE) --> [click here](#LICENSE)
+            .replace(
+              /\[([^[\]]*)\]\(([^)#]+)\)/g,
+              (match, textContent, href) => `[${textContent}](${urlModule.resolve(url, href)})`
+            )
+            // Modify links to keep them within the site
+            .replace(
+              /https?:\/\/github.com\/(webpack|webpack-contrib)\/([-A-za-z0-9]+-loader\/?)([)"])/g,
+              '/loaders/$2/$3'
+            )
+            .replace(
+              /https?:\/\/github.com\/(webpack|webpack-contrib)\/([-A-za-z0-9]+-plugin\/?)([)"])/g,
+              '/plugins/$2/$3'
+            )
+            // Replace any <h2> with `##`
+            .replace(/<h2[^>]*>/g, '## ')
+            .replace(/<\/h2>/g, '')
+            // Drop any comments
+            .replace(/<!--[\s\S]*?-->/g, '');
+        }
+
+        var title = pkg.name;
+        if (title.match(/-plugin$/)) {
+          title = _.camelCase(title);
+          title = _.upperFirst(title);
+          title = title.replace(/I18N/, 'I18n');
+        }
+
+        // TODO: push this type of stuff to a script of its own to keep this standard
+        let headmatter = yamlHeadmatter({
+          title: title,
+          source: url,
+          edit: [pkg.html_url, 'edit', branch, file].join('/'),
+          repo: pkg.html_url
+        });
+
+        return async.parallel(
+          [
+            fs.writeFile.bind(null, path.resolve(options.output, pkg.name + path.extname(file)), headmatter + body),
+            fs.writeFile.bind(null, path.resolve(options.output, pkg.name + '.json'), JSON.stringify(pkg, null, 2))
+          ],
+          function(err) {
+            if (err) {
+              return cb(err);
+            }
+
+            return cb(null, pkg);
+          }
+        );
+      });
+    },
+    finalCb
+  );
+}
+
+// TODO: push this type of to a script of its own to keep this generic
+function yamlHeadmatter(fields) {
```

Comment:
Don't we have a util that already generates this based on a set of information? I think I did separate this logic, would love if you could adapt it so we don't go back to tech debt around this;

Reply:
will try

Reply:
@jeremenichelli `rebuild` branch probably, it is not in `master`

Reply:
Oh sorry I thought this was on the rebuild one. Fine then :)